source/administration/services/hdbpp/hdbpp-cm-interface.rst

```diff
@@ -0,0 +1,152 @@
+.. _hdbpp_cm-interface:
+
+:audience:`administrators, developers`, :lang:`C++`
+
+
+Configuration Manager interface
+-------------------------------
+
+.. contents::
+   :depth: 2
+
+.. Configuration Manager interface
+
+More in detail the device server exposes the following interface.
+
+Commands
+~~~~~~~~
+
+The commands availabile in the are summarized in commands-table.
```

Comment:
Suggest to rephrase: "The commands available are summarized in commands-table."

Reply:
Thanks, done